data/tools/wesnoth/wmltools3.py

```diff
@@ -267,7 +283,7 @@ def __iter__(self):
 
 def iswml(filename):
     "Is the specified filename WML?"
-    return filename.endswith(".cfg")
+    return filename.endswith(default_wml_extension)
```

Comment:
Isn't this supposed to look the same as the ismap function?

Reply:
Updated.